final/src/components/Dashboard1/ListItem.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ListItem = props => {
+  const { close, time } = props.item;
+  const { lastItem, currency } = props;
+
+  let change = 0; //default value;
+  if (lastItem && close) {
+    change = (((close - lastItem.close) / close) * 100).toFixed(2);
+    console.log(close, lastItem.close);
+  }
+
+  const date = new Date(time * 1000);
+  const fyear = date.getFullYear();
+  const fmonth =
+    date.getMonth() + 1 < 10
+      ? '0' + (date.getMonth() + 1)
+      : date.getMonth() + 1;
+  const fday = date.getDate() < 10 ? '0' + date.getDate() : date.getDate();
+  const fhours = date.getHours() < 10 ? '0' + date.getHours() : date.getHours();
+  const fminutes =
+    date.getMinutes() < 10 ? '0' + date.getMinutes() : date.getMinutes();
+
+  const formattedDate = `${fyear}-${fmonth}-${fday} ${fhours}:${fminutes}`;
```

Comment:
Why not to use moment.js for date formatting. In js is a good practise to use moment for working with dates.